doc/internal/ReleaseInstructions.md

```diff
@@ -95,137 +114,175 @@ Install `gpg-agent` (needed below) e.g. on Ubuntu via: `sudo apt-get install gnu
+
+This step happens a few weeks before the actual release (whether it is an RC, GA or a patch release).
+The main goal of this step is to make sure everything is ready to be released for the release day.
+That includes:
+- **Making sure Pull Requests are being reviewed and merged.**
+  > - All the Pull Requests that needs to be in the release must be reviewed and merged before the code freeze.
+  > - The code freeze usually happens a few days before the release. Make sure everything is merged by that date.
+- **Making sure the people doing the release have access to all the tools and infrastructure needed to do the release.**
+  > - This includes write access to the Vitess repository and to the Maven repository. 
+- **Preparing and cleaning the release notes summary.**
+  > - One or more Pull Requests have to be submitted in advance to create and update the release summary.
+  > - The summary files are located in: `./doc/releasenotes/*_*_*_summary.md`.
+  > - The summary file for a release candidate is the same as the one for the GA release.
+- **Finishing the blog post, and coordinating with the different organizations for cross-posting. Usually CNCF and PlanetScale. This step applies only for GA releases.**
+  > - The blog post must be finished and reviewed.
+  > - A Pull Request on the website repository of Vitess has to be created so we can easily publish the blog during the release day.
+- **Code freeze.**
+  > - As soon as we go into code freeze, if we are doing an RC, create the release branch.
+  > - If we are doing a GA release, do not merge any new Pull Requests.
+  > - The guide on how to do a code freeze is available in the [How To Code Freeze](#how-to-code-freeze) section.
+- **Preparing the Vitess Operator release.**
+  > - While the Vitess Operator is located in a different repository, we also need to do a release for it.
+  > - The Operator follows the same cycle: RC1 -> GA -> Patches.
+  > - A documentation for the pre-release of the Vitess Operator is available [here](https://github.com/planetscale/vitess-operator/blob/main/docs/release-process.md#prepare-for-release).
+
+### Release
+
+On the release day, there are several things to do:
+
+- **Create the Vitess release.**
+  > - A guide on how to create a Vitess release is available in the [How To Release Vitess](#how-to-release-vitess) section.
+- **Create the corresponding Vitess operator release.**
+  > - Applies only to versions greater or equal to `v14.0.0`.
+  > - If we are doing an RC release, then we will need to create the Vitess Operator RC too. If we are doing a GA release, we're also doing a GA release in the Operator.
+  > - The Vitess Operator release documentation is available [here](https://github.com/planetscale/vitess-operator/blob/main/docs/release-process.md#cut-release).
+- **Create the Java release.**
+  > - Applies only to GA releases.
+  > - This step is explained in the [Java Packages Deploy & Release](#java-packages-deploy--release) section.
```

Comment:
```suggestion
  > - This step is explained in the [Java Packages: Deploy & Release](#java-packages-deploy--release) section.
```

Reply:
Not sure if the link will still work after introducing a `:`. You will need to verify that.

Reply:
This syntax seems to work